doc/source/math/basic.rst

```diff
@@ -63,11 +63,22 @@ constants about the universe that you may find handy in your math operations.
 
 .. global:: Constant:PI
 
-    Ratio of circumference of a circle to its diameter
+    Ratio of circumference of a circle to its diameter, 3.14159265...
+    
+    ::
+
+        SET diameter to 10.
+        PRINT "circumference is:".
+        PRINT constant:pi * diameter.
```

Comment:
1st off, I don't think it's a good idea for us to be exclusionary and imply that if a person doesn't understand the definition of pi they shouldn't use kOS.  KSP itself caters to all kinds of people, including some surprisingly young children.  It isn't hard to believe that some one might get their start with kOS only using wait commands to stage or throttle.  But that doesn't mean it's our job to explain pi, just that we should be careful about the phrasing.

On topic however: if it's important enough to mention in the text explanation I'm not sure that we can argue against providing a code example as well.  At least the example is _doing_ something with the value.

Remove the example or leave it, I don't really mind either way, just wanted to add my two cents.